gfx.cpp

```diff
@@ -2788,6 +2844,9 @@ class GfxInternal
     {
         if(isInterop())
             return GFX_SET_ERROR(kGfxResult_InvalidOperation, "Cannot clear backbuffer when using an interop context");
+        // No swap chain - no back buffer
+        if(!swap_chain_)
+            return kGfxResult_NoError;
```

Comment:
Similarly to trying to clear the backbuffer from an "interop context", this should return an error to the user rather than failing/nop-ing silently.